common/scripted_buttons/imperia_dead_man_buttons.txt

```diff
@@ -0,0 +1,172 @@
+﻿tur_republican_button = {
+	name = "republican_button"
+	desc = "republican_button_desc"
+
+	visible = {
+		owner = { 
+			has_variable = chose_initial_turk_government 
+		}
+	}
+
+	possible = {
+		trigger_if = {
+			limit = {
+				has_variable = has_selected_republican_once_var
+			}
+			custom_tooltip = {
+				text = cannot_choose_republican_twice_tt
+				always = no
+			}
+		}
+		any_interest_group = {
+			is_in_government = yes
+			OR = {	
+				leader = {
+					OR = {
+						has_ideology = ideology:ideology_republican_leader
+						has_ideology = ideology:ideology_radical
+						has_ideology = ideology:ideology_atheist # nihilist
+						has_ideology = ideology:ideology_liberal_leader #democrat
+						has_ideology = ideology:ideology_positivist
+						has_ideology = ideology:ideology_market_liberal
+					}
```

Comment:
This and the following triggers should also be their respective custom triggers, no?